### ISSUE TITLE
SpatialDropout2D

### ISSUE THREAD

**Issue by crockpotveggies** [deeplearning4j/deeplearning4j#5109]
#### Issue Description

Attempted to import this model of U-Net and received the following error:

```
Exception in thread "main" org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unsupported keras layer type SpatialDropout2D. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:277)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:196)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:158)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:91)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:135)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:145)
	at org.deeplearning4j.nn.modelimport.keras.KerasConverter.convertKerasFunctional(KerasConverter.java:22)
	at org.deeplearning4j.nn.modelimport.keras.KerasConverter.main(KerasConverter.java:47)
```

@maxpumperla I will DM you the file.

#### Version Information

snapshot

#### Contributing

If you'd like to help us fix the issue by contributing some code, but would
like guidance or help in doing so, please mention it!

**Comment by maxpumperla** [comment]
@AlexDBlack not sure if we can manage to squeeze this in before release, but it seems @crockpotveggies needs this for an interesting app in the zoo. Not sure if you know spatial dropout, but it's based on this paper https://arxiv.org/abs/1411.4280.

I'm basically OOO for the rest of the week, but if _maybe_ you can start implementing the DL4J layer, I could sneak in keras import until Friday.

**Comment by AlexDBlack** [comment]
Added for DL4J (but not Keras import) here: https://github.com/deeplearning4j/deeplearning4j/pull/5117

**Comment by maxpumperla** [comment]
@crockpotveggies good to go
